lib/patches/rspec-support/rspec/support/ruby_features.rb

```diff
@@ -0,0 +1,12 @@
+Truffle::Patching.require_original __FILE__
+
```

Comment:
It appears newer rspec moves this method to a new file.

Reply:
Could you document the versions in the patch files. I think being cautions and fail when the file tries to patch a wrong version safes us troubleshooting. Could the old patch be removed?

Reply:
It actually didn't move files. I think the current patch is in `support.rb`probably due to rspec's different require system. I've just removed the `ruby_features.rb`change from this pull request for now to figure out at another time/in another pull request.